### PR TITLE
[Documentation] Enhance descriptions of chapters in the Book + index

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Doc fix?        | yes |
| New docs?    | no |
| Related tickets |  |
| License         | MIT |

### CODE REVIEW

File: docs/book/index.rst [hunk 1]
```diff
@@ -4,15 +4,93 @@ The Book
...
+
     products/index
+
+.. include:: /book/products/map.rst.inc
+
+Orders
+------
+
+In this chapter you will learn everything you need to know about orders in Sylius.
+This concepts comes together with a few additional ones, like promotions, payments, shipments or checkout in general.
```
Comment (GSadee) [inline docs/book/index.rst]: `concepts` -> `concept`

File: docs/book/index.rst [hunk 2]
```diff
@@ -4,15 +4,93 @@ The Book
...
     :hidden:
 
     introduction/index
+
+.. include:: /book/introduction/map.rst.inc
+
+Installation
+------------
+
+The installation chapter is of course a comprehent guide to installing Sylius on your machine, but it also provides
```
Comment (lchrusciel) [inline docs/book/index.rst]: `comprehend`?

Comment (TheMadeleine) [inline docs/book/index.rst]: :rofl: `comprehensive` of course :P

**Comment by pjedrzejewski** [comment]
Nice, thank you Magda! :)

**Pull request merged**